tests/ImageSharp.Tests/Drawing/DrawImageTests.cs

```diff
@@ -71,49 +70,46 @@ public void ImageBlendingMatchesSvgSpecExamples<TPixel>(TestImageProvider<TPixel
             float opacity)
             where TPixel : struct, IPixel<TPixel>
         {
-            using (Image<TPixel> image = provider.GetImage())
-            using (var blend = Image.Load<TPixel>(TestFile.Create(brushImage).Bytes))
+            using Image<TPixel> image = provider.GetImage();
+            using var blend = Image.Load<TPixel>(TestFile.Create(brushImage).Bytes);
+            var size = new Size(image.Width * 3 / 4, image.Height * 3 / 4);
+            var position = new Point(image.Width / 8, image.Height / 8);
+            blend.Mutate(x => x.Resize(size.Width, size.Height, KnownResamplers.Bicubic));
+            image.Mutate(x => x.DrawImage(blend, position, mode, opacity));
+            FormattableString testInfo = $"{System.IO.Path.GetFileNameWithoutExtension(brushImage)}-{mode}-{opacity}";
+
+            var encoder = new PngEncoder();
+
+            if (provider.PixelType == PixelTypes.Rgba64)
             {
-                var size = new Size(image.Width * 3 / 4, image.Height * 3 / 4);
-                var position = new Point(image.Width / 8, image.Height / 8);
-                blend.Mutate(x => x.Resize(size.Width, size.Height, KnownResamplers.Bicubic));
-                image.Mutate(x => x.DrawImage(blend, position, mode, opacity));
-                FormattableString testInfo = $"{System.IO.Path.GetFileNameWithoutExtension(brushImage)}-{mode}-{opacity}";
-
-                var encoder = new PngEncoder();
-
-                if (provider.PixelType == PixelTypes.Rgba64)
-                {
-                    encoder.BitDepth = PngBitDepth.Bit16;
-                }
-
-                image.DebugSave(provider, testInfo, encoder: encoder);
-                image.CompareToReferenceOutput(ImageComparer.TolerantPercentage(0.01f),
-                    provider,
-                    testInfo);
+                encoder.BitDepth = PngBitDepth.Bit16;
             }
+
+            image.DebugSave(provider, testInfo, encoder: encoder);
+            image.CompareToReferenceOutput(
+                ImageComparer.TolerantPercentage(0.01f),
+                provider,
+                testInfo);
         }
 
         [Theory]
-        [WithTestPatternImages(200, 200, PixelTypes.Rgba32 | PixelTypes.Bgra32)]
+        [WithTestPatternImage(200, 200, PixelTypes.Rgba32 | PixelTypes.Bgra32)]
```

Comment:
The plural form might make sense here, since the attribute is feeding an image for each requested pixel type.

Reply:
changed with: f897ab6f8da1c3efe770039dd0d9d41ad879afb3